tests/golden/defaults-with-provider/crossplane/crossplane/01_helmchart/crossplane/templates/clusterrole.yaml

```diff
@@ -12,8 +12,8 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: crossplane
     app.kubernetes.io/part-of: crossplane
-    app.kubernetes.io/version: 1.9.0
-    helm.sh/chart: crossplane-1.9.0
+    app.kubernetes.io/version: 1.10.0
```

Comment:
Any reason why we haven't gone directly with version 1.11?

Reply:
as far as I know, the latest stable is 1.10

Reply:
Yes you are right, though there's a helm chart 1.11 already which has a `rc` on crossplane 1.11. It's a bit confusing